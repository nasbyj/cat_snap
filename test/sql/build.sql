\set ECHO none

\i test/pgxntool/psql.sql
\i test/pgxntool/tap_setup.sql

BEGIN;
\i sql/cat_snap.sql

SET LOCAL search_path = tap, cat_snap;

SELECT plan(2);

SELECT hasnt_schema('pg_temp');
SELECT hasnt_schema('__cat_snap');

SELECT finish();

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
