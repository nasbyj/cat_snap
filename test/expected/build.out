\set ECHO none
13
1..2
ok 1 - Schema pg_temp should not exist
ok 2 - Schema __cat_snap should not exist
# TRANSACTION INTENTIONALLY LEFT OPEN!
